diff --git a/grails-app/views/public/showDataResource.gsp b/grails-app/views/public/showDataResource.gsp
index 936d1c8..c42f89e 100644
--- a/grails-app/views/public/showDataResource.gsp
+++ b/grails-app/views/public/showDataResource.gsp
@@ -38,7 +38,7 @@
                     <h2 class="dataResourceProviderLink"><g:link action="show" id="${dp.uid}">${dp.name}</g:link></h2>
                 </g:if>
                 <g:if test="${instance.institution}">
-                    <h2 class="dataResourceInstitutionLink"><g:link action="show" id="${instance.institution.uid}">${instance.institution.name}</g:link></h2>
+                    <h3 class="dataResourceInstitutionLink"><g:link action="show" id="${instance.institution.uid}">${instance.institution.name}</g:link></h3>
                 </g:if>
             </div>
 
